program/rust/src/processor/upd_price.rs

```diff
@@ -123,7 +123,7 @@ pub fn upd_price(
     }?;
 
     check_valid_funding_account(funding_account)?;
-    check_valid_writable_account(program_id, price_account)?;
+    check_valid_writable_account_without_rent_check(program_id, price_account)?;
```

Comment:
mmmmm what about removing the rent check everywhere? I think the check is not useful anywhere

Reply:
this is introducing more complexity